commit 249e03b99266d33311aabbe638c5d2789e5475a9 (refs/changes/11/811/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-10-28T17:59:04+00:00 (2 years, 11 months ago)
    
    MORAY-369 Document that 'retries' cannot be Infinity with node-moray v2

diff --git a/CHANGES.md b/CHANGES.md
index 6dbd88e..5aa5991 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -21,6 +21,10 @@ networking failures.
   updated to use
   [VError.findCauseByName](https://github.com/joyent/node-verror#verrorfindcausebynameerr-name)
   instead of checking the `name` property directly.**
+* `retry.retries` must now be a finite number. Once the number of retries has
+  been exceeded for a backend, the "cueball" module will consider that backend
+  dead, and will take care of periodically checking it to see if it's returned.
+  The recommended and default value is `5`.
 
 **Other changes:**
 
diff --git a/package.json b/package.json
index 89577af..9150387 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
     "description": "Moray client library",
     "version": "2.0.0",
     "author": "Joyent (joyent.com)",
+    "keywords": [ "moray" ],
     "main": "./lib/index.js",
     "repository": {
         "type": "git",
